src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1

```diff
@@ -160,6 +160,17 @@ function script:chocoRemotePackages($filter) {
         ForEach-Object { $_.Split('|')[0] }
 }
 
+function script:chocoRemotePackageVersions($Name, $Version) {
+        $packageVersions = & $script:choco search $Name --exact --all-versions --limit-output |

```

Comment:
Not sure if this will extend to other parts of tab completion beyond this PR, but since we now have `--include-headers`, we should be able to use it here.

Reply:
That is fair, I didn't think of using it.

I think this is the only place where we do the easy way out to convert it from CSV.

I'll look at updating this method.